src/Rings/AbelianClosure.jl

```diff
@@ -150,22 +150,33 @@ parent(::QQAbElem{NfAbsNSElem}) = _QQAb_sparse
 #
 ################################################################################
 
-_variable(K::QQAbField) = K.s
+function _variable(K::QQAbField)
+  if isdefined(K, :s)
+    return K.s
+  else
+    if Oscar.is_unicode_allowed()
+      return "ζ"
```

Comment:
If we want to avoid the explicit `"ζ"`  in the code then we can also write `transcode(String, [UInt16(950)])`.
(The `950` can be computed as `Int(codepoint('ζ'))`.)

Reply:
I think this should be fine as-is. At least we have similar things in other places.